### DISCUSSION TITLE
Two-factor SMS support for Turkey

### DISCUSSION THREAD

**Discussion by bahadirhanfiliz** [community/community#59755] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why Turkey is not included in the supported counties for two-factor SMS verification?

There are a lot of developers from Turkey who contribute and use Github services. 

Please consider adding SMS authentication support for Turkey.

**Comment by mhsabbagh** [comment]
GitHub now requires 2FA for all accounts, but won't allow users in Turkey to receive verification via SMS. Wow.

I hope it is added soon.

**Comment by Alpho320** [comment]
pump, please do someting about this.

**Comment by ghostinhershell** [comment]
Hi @bahadirhanfiliz, 

Thanks so much for reaching out in our community. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

Let me know if you have any additional questions on this topic!

**Comment by cetindogu** [comment]
i have been waiting SMS auth  for Türkiye    +90 are code

**Comment by irsat000** [comment]
I see Uganda, Nigeria, Uruguay, where even is Uruguay? And no Turkey, wow.

**Comment by Winvoker** [comment]
Is there any updates on this? Are you even planning? @ghostinhershell

**Comment by bisultankishiev** [comment]
[NY Lead Generation Experts](https://pearllemonleadsusa.com/lead-generation-agency-new-york/) know the importance of secure authentication Turkey’s developers deserve SMS verification too! Hope GitHub expands support soon.

**Comment by basrioglumehmet** [comment]
ADD SUPPORT PLS

**Comment by turgutguvercin** [comment]
2026 still no sms 2FA for Turkey

**Comment by byhipernova** [comment]
Ulan alakasız ülkeler bile var Türkiyeyi de eklesenize
